oximeter/oximeter/schema/switch-data-link.toml

```diff
@@ -164,6 +173,51 @@ versions = [
     { added_in = 1, fields = [ "port_id", "link_id" ] }
 ]
 
+[[metrics]]
+name = "fec_high_symbol_errors"
+description = "FEC symbol error threshold exceeded"
+units = "none"
+datum_type = "bool"
+versions = [
+    { added_in = 1, fields = [ "port_id", "link_id" ] }
+]
+
+[[metrics]]
+name = "fec_aligned"
+description = "All lanes synchronized and aligned"
+units = "none"
+datum_type = "bool"
+versions = [
+    { added_in = 1, fields = [ "port_id", "link_id" ] }
+]
+
+[[metrics]]
+name = "fec_corrected_blocks"
+description = "Total number of FEC blocks that were corrected"
+units = "count"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [ "port_id", "link_id" ] }
+]
+
+[[metrics]]
+name = "fec_uncorrected_blocks"
+description = "Total number of FEC blocks that were uncorrected"
+units = "count"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [ "port_id", "link_id" ] }
+]
+
+[[metrics]]
+name = "fec_symbol_errors"
+description = "Total number of FEC symbol errors"
+units = "count"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [ "port_id", "lane", "link_id" ] }
```

Comment:
@bnaecker do we only want to pick up the lane (meta)data on errors vs other fec-related metrics?

Reply:
These timeseries are populated by counters provided by the ASIC and the Tofino SDE. They only provide per-lane breakdowns for FEC symbol errors. Things like high error rates and the like are for the entire link, rather than per-lane unfortunately.